sql/planbuilder/scope.go

```diff
@@ -42,6 +41,8 @@ type scope struct {
 	windowDefs  map[string]*sql.WindowDefinition
 	// exprs collects unique expression ids for reference
 	exprs map[string]columnId
+	// subqueries is a workaround for execution limitations
+	subqueries map[string]scopeColumn
```

Comment:
Name is not great, unclear what a field called subqueries with this type could be. Missing a noun or two.

Reply:
deleting, this was from a discarded prototype